src/world/area_kzn/kzn_02/C5AA50.c

```diff
@@ -1,9 +1,45 @@
 #include "kzn_02.h"
 
-INCLUDE_ASM(s32, "world/area_kzn/kzn_02/C5AA50", func_80240310_C5AA50);
+ApiStatus func_80240310_C5AA50(Evt* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    f32 sp20;
+    f32 sp24;
+    f32 sp28;
+    f32 sp2C;
+    f32 sp30;
+    f32 sp34;
+
+    get_model_center_and_size(evt_get_variable(script, *args++), &sp20, &sp24, &sp28, &sp2C, &sp30, &sp34);
```

Comment:
Forgot to mention this, but check this function's prototype, if the args are named (e.g. something more descriptive than arg0, etc.) then you can name your vars the same here (e.g. sp20 and co would have the same names as the arguments)